pkg/asset/imagebased/configimage/configiso.go

```diff
@@ -0,0 +1,134 @@
+package configimage
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+
+	"github.com/openshift/assisted-image-service/pkg/isoeditor"
+	"github.com/openshift/installer/pkg/asset"
+	agentAsset "github.com/openshift/installer/pkg/asset/agent"
+)
+
+const (
+	configImageFilename = "imagebasedconfig.noarch.iso"
+	configImageLabel    = "imagebased_configimage"
+	configCpioArchive   = "config.gz"
```

Comment:
Where is this used?

Reply:
Good catch, it's not used anywhere, I'll remove it :+1: